frontend/packages/console-app/src/components/cloud-shell/CloudShellTerminal.tsx

```diff
@@ -44,25 +87,36 @@ const CloudShellTerminal: React.FC<CloudShellTerminalProps> = ({ username, onCan
     return <LoadingBox />;
   }
 
-  if (Array.isArray(data)) {
-    const workspace = data.find(
-      (d) => d?.metadata?.annotations?.[CLOUD_SHELL_USER_ANNOTATION] === username,
+  if (!workSpacePod && initializing) {
+    return (
+      <div className="odc-cloudshell-terminal__container">
+        <LoadingBox message="Connecting you to OpenShift command line terminal" />;
```

Comment:
Why are you changing the message from the previous only documented in the design doc?